attributes/default.rb

```diff
@@ -16,7 +16,7 @@
   default['repose']['repo']['baseurl'] = 'http://repo.openrepose.org/debian'
   default['repose']['repo']['gpgkey'] = 'http://repo.openrepose.org/debian/pubkey.gpg'
   default['repose']['repo']['managed'] = true
-  default['repose']['install_opts'] = '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes'
+  default['repose']['install_opts'] = '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --allow-downgrades --allow-change-held-packages --yes'
```

Comment:
I'm not entirely sure what's going on here. Did the repose packages get upgraded by accident, causing the apt operation to fail? Are they held? They probably should be so that they don't get upgraded at random. In any case, this change is probably ok, but I'm thinking there might be other things we need to do.